packages/components/src/FormField/components/ClearAction.css

```diff
@@ -1,12 +1,6 @@
-.clearContainer {
-  display: flex;
-  align-items: center;
-  padding-left: var(--space-small);
-}
-
 .clearInput {
   display: flex;
-  position: relative;
+  position: absolute;
```

Comment:
I noticed something with clearable on long text in a single line input:
On this branch:
<img width="1530" alt="image" src="https://github.com/GetJobber/atlantis/assets/17303106/cfa19ab0-3919-4e82-8388-d35feb6ce52f">

On master:
<img width="1526" alt="image" src="https://github.com/GetJobber/atlantis/assets/17303106/04bab422-e7cf-47e7-8f95-0f205413876b">


I also noticed the Clearable will overlap with the loading if they are both present (not this happens even if the toolbar prop isn't used).
<img width="1529" alt="image" src="https://github.com/GetJobber/atlantis/assets/17303106/f1dbbd20-0d15-4575-ab38-f72f44fd3b2f">

Reply:
I also noticed this when the InputText is wrapped in a div with a max width of 50%. I found changing this absolute to relative "fixed" it but I haven't fully caught up on the context of why that change was made so there is probably other things to consider
<img width="1528" alt="image" src="https://github.com/GetJobber/atlantis/assets/17303106/09bc5642-d568-4013-b27f-eee54de22fe3">

Reply:
More for me to test afterwards but this also happens with InputPassword's that are clearable and have visibility enabled
<img width="1534" alt="image" src="https://github.com/GetJobber/atlantis/assets/17303106/4bfaba8e-e844-443e-9304-55e48de5693b">

InputNumber and the browser default stepper on firefox (may be the same fix in general)
<img width="1538" alt="image" src="https://github.com/GetJobber/atlantis/assets/17303106/5bbe855d-6d68-4d25-a5b8-69bf9c8a3529">

Reply:
Also focused with a suffix can overlap (the same as InputPassword but more general. This also happens with Icon
<img width="1535" alt="image" src="https://github.com/GetJobber/atlantis/assets/17303106/b1428316-ca95-49df-9604-f31cd177741a">

Reply:
ugh yeah I see what you mean

clearable being positioned absolutely is because previously it would center itself vertically, and that worked because it was only ever as tall as the single line input. now that the single input can be taller depending on whatever you've given it as a tool bar, we want it to be top right - not centered on the overall height

worth nothing clearable + multiline is broken on master right now. that and a bunch of other props (prefix, suffix, loading) don't play very well with multiline. unfortunately the underlying reason they don't work is because of height and toolbar adds height 😭 even to the single line

that long input thing happens today with loading, because it's absolutely positioned so yeah that tracks that it now happens with clearable

<img width="644" alt="image" src="https://github.com/GetJobber/atlantis/assets/11843215/103ec9a7-61a0-43c9-a55b-392e838d3858">

Reply:
overall the problem is that so many of these things (suffix, affix, loading, clearable, password eye thing) are all built assuming a set height

once that height changes, like it does with the toolbar, they break.

they shouldn't be centered, they should be top right/top left -ish and that's where the absolute comes in, but yeah as we're seeing that has a lot of side effects

Reply:
master, InputNumber + loading

you can barely see it but yeah that same overlapping control problem

<img width="643" alt="image" src="https://github.com/GetJobber/atlantis/assets/11843215/0a429c4d-d182-4adf-b522-2ddd2b756632">